assets/datagrid.js

```diff
@@ -643,11 +643,7 @@ $(document).on('click', '[data-datagrid-editable-url]', function(event) {
 			input.attr('rows', Math.round(cell_lines));
 		} else if (cell.data('datagrid-editable-type') === 'select') {
 			input = $(cell.data('datagrid-editable-element'));
-			input.find('option').each(function() {
-				if ($(this).val() === valueToEdit) {
```

Comment:
this is probably a problem with numeric options

Reply:
Why is that?

Reply:
This is wrong because there is strict comparsion of type, but teher we need "1" == 1. And this notation is unnecessarily long. `input.find("option[value='" + valueToEdit + "']")` fixed this.